src/main/java/io/percy/appium/lib/CliWrapper.java

```diff
@@ -95,7 +95,7 @@ public String postScreenshot(String name, JSONObject tag, List<Tile> tiles, Stri
         data.put("externalDebugUrl", externalDebugUrl);
         data.put("ignoredElementsData", ignoredElementsData);
         data.put("consideredElementsData", consideredElementsData);
-        data.put("clientInfo", env.getClientInfo());
+        data.put("clientInfo", env.getClientInfo(false));
```

Comment:
we should not be updating this right? this will show up in UI

Reply:
Yes, if you look the `getClientInfo`. passing false will not change anything.